src/lib/_common.scss

```diff
@@ -117,34 +118,51 @@ $header-height: 48px;
     }
 }
 
-.bp {
-    align-items: center;
-    background-color: $ffive;
-    border: 0 none;
+.bp-body {
     bottom: 0;
     display: flex;
-    flex-direction: column;
-    justify-content: center;
+    flex-direction: row;
     left: 0;
-    margin: 0;
-    outline: none;
-    -webkit-overflow-scrolling: touch;
-    padding: 0;
     position: absolute;
     right: 0;
     top: 0;
 
-    // Pseudo fullscreen for iOS Safari which doesn't support the fullscreen API
-    &.bp-fullscreen-unsupported {
+    .bp-thumbnails-container {
+        background-color: $ffive;
+        border-right: solid 1px $seesee;
+        width: 180px;
+    }
+
+    .bp {
```

Comment:
Is changing the markup structure here considered a breaking change? If so, we may want to keep the outer container as `.bp` and rename the inner container to `.bp-preview` or something for clarity. Some CSS frameworks advocate for using hyphens as descendent indicators, so it'd be `.bp` -> `.bp-preview`.

Reply:
Ok, as discussed, I'll introduce a Upgrade.md describing the breaking change. Also I'll make the following changes:
`.bp-body` -> `.bp`
`.bp` -> `.bp-content`